### PR TITLE
uischema: add missing ui options

### PR DESCRIPTION
**Pull request by yeze322**
Fixes #

## Description

Add missing UI options in *.uischema files. Fixes several issues in Composer.

Closes https://github.com/microsoft/BotFramework-Composer/issues/7034
Closes https://github.com/microsoft/BotFramework-Composer/issues/7035

## Specific Changes

  - Add missing `menu` uischema for built-in actions (duplicated from Composer, Composer will remove those hardcoded uischema)
  - Add missing `menu` uischema for several actions mentioned in https://github.com/microsoft/BotFramework-Composer/issues/7034
  - Add missing `trigger` uischema for triggers mentioned in https://github.com/microsoft/BotFramework-Composer/issues/7035

### CODE REVIEW

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.DateTimeInput.uischema [hunk 1]
```diff
@@ -12,6 +12,10 @@
             }
         }
     },
+    "menu": {
+        "label": "Date or time",
```
Comment (chrimc62) [inline libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.DateTimeInput.uischema:16]: >or [](http://example.com/codeflow?start=23&length=2)

and/or

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.OAuthInput.uischema [hunk 2]
```diff
@@ -9,6 +9,16 @@
...
     },
+    "menu": [
+        {
+            "label": "OAuth login",
+            "submenu": ["Ask a question"]
+        },
+        {
+            "label": "OAuth login",
+            "submenu": ["Access external resources"]
+        }
```
Comment (chrimc62) [inline libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.OAuthInput.uischema:20]: You mean to put it in multiple places?

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnContinueConversation.uischema [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://schemas.botframework.com/schemas/ui/v1.0/ui.schema",
+  "form": {
+      "hidden": [
+          "actions"
+      ]
+  },
+  "trigger": {
+    "submenu": "Activities"
```
Comment (chrimc62) [inline libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnContinueConversation.uischema:9]: No label?

**Review by chrimc62 — APPROVED** [review]
:shipit:

**Review by luhan2017 — APPROVED** [review]

**Comment by luhan2017** [comment]
@mrivera-ms Could you please help to merge it? I don't have the permission.

**Review by mrivera-ms — APPROVED** [review]

**Pull request merged**